CMakeLists.txt

```diff
@@ -362,7 +380,11 @@ endif ()
 target_link_libraries(${VSOMEIP_NAME}-e2e ${VSOMEIP_NAME} ${Boost_LIBRARIES} ${USE_RT} ${DL_LIBRARY} ${SystemD_LIBRARIES})
 
 if(${CMAKE_SYSTEM_NAME} MATCHES "QNX")
+    target_link_directories(${VSOMEIP_NAME} PUBLIC "${QNX_TARGET}/${CPUVARDIR}/io-sock/lib")
```

Comment:
I'm not sure how well this generalizes to different QNX deployments.  I would recommend that QNX linker paths be handled by a toolchain managed by the user (outside of this repo)